vector/src/main/java/im/vector/riotx/features/crypto/quads/SharedSecuredStoragePassphraseFragment.kt

```diff
@@ -41,15 +46,17 @@ class SharedSecuredStoragePassphraseFragment : VectorBaseFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        ssss_restore_with_passphrase_warning_text.text = span {
-            span(getString(R.string.enter_secret_storage_passphrase_warning)) {
-                textStyle = "bold"
-            }
-            +" "
-            +getString(R.string.enter_secret_storage_passphrase_warning_text)
-        }
-
-        ssss_restore_with_passphrase_warning_reason.text = getString(R.string.enter_secret_storage_passphrase_reason_verify)
+        // If has passphrase
+        val pass = getString(R.string.recovery_passphrase)
+        val key = getString(R.string.recovery_key)
+        ssss_restore_with_passphrase_warning_text.text = getString(
+                R.string.enter_secret_storage_passphrase_or_key,
+                pass,
+                key
+        )
+                .toSpannable()
+                .colorizeMatchingText(pass, colorProvider.getColorFromAttribute(android.R.attr.textColorLink))
+                .colorizeMatchingText(key, colorProvider.getColorFromAttribute(android.R.attr.textColorLink))
```

Comment:
links are still not clickable, to it confuses users. See #1257 . Maybe comment this out for the moment?

Reply:
There will be a link at some point